apps/betterangels-backend/notes/tests/test_queries.py

```diff
@@ -192,57 +192,121 @@ def test_notes_paginated_query(self) -> None:
         self.assertFalse(note_differences)
 
     @parametrize(
-        (
-            "case_manager_label, client_label, search_terms, "
-            "is_submitted, expected_results_count, "
-            "returned_note_label_1, returned_note_label_2"
-        ),
+        ("case_manager_label, client_label, is_submitted, expected_results_count, returned_note_labels"),
         [
             # Filter by:
-            # created by, client_label, search terms, and/or is_submitted
-            ("org_1_case_manager_1", None, None, None, 1, "note", None),  # CM 1 created one note
-            ("org_1_case_manager_2", None, None, None, 2, "note_2", "note_3"),  # CM 2 created 2 notes
-            ("org_1_case_manager_1", None, "deets", None, 0, None, None),  # None of CM 1's notes contain "deets"
-            ("org_1_case_manager_2", None, "deets", None, 2, "note_2", "note_3"),  # Two of CM 2's notes contain "deets"
-            # CM 2 has one note "deets" for client "truman"
-            ("org_1_case_manager_2", None, "deets rum", None, 1, "note_3", None),
-            ("org_1_case_manager_2", None, "deets rum", True, 0, None, None),  # CM 2 has no submitted notes
-            ("org_1_case_manager_1", "client_user_2", None, None, 0, None, None),  # CM 1 has no notes for client 2
-            # CM 2 has one unsubmitted note for client 1
-            ("org_1_case_manager_2", "client_user_1", None, False, 1, "note_2", None),
-            (None, None, None, True, 0, None, None),  # There are no submitted notes
-            (None, None, None, None, 3, False, None),  # There are three unsubmitted notes
+            # created by, client_label, and/or is_submitted
+            ("org_1_case_manager_1", None, None, 1, ["note"]),  # CM 1 created one note
+            ("org_1_case_manager_2", None, None, 2, ["note_2", "note_3"]),  # CM 2 created 2 notes
+            ("org_1_case_manager_2", None, False, 1, ["note_2"]),  # CM 2 has one unsubmitted note
+            ("org_1_case_manager_2", None, True, 1, ["note_3"]),  # CM 2 has one submitted note
+            ("org_1_case_manager_1", "client_user_2", None, 0, []),  # CM 1 has no notes for client 2
+            ("org_1_case_manager_1", "client_user_1", False, 1, ["note"]),  # CM 1 has one unsubmitted note for client 1
+            (None, None, False, 2, ["note", "note_2"]),  # There are two unsubmitted notes
         ],
     )
     def test_notes_query_filter(
```

Comment:
i know this misses is submitted but i think it's ok
```suggestion
    def test_notes_query_filter_by_users(
```

Reply:
I plan to rename this test but not sure to what. There's a ticket to add an orgs filter and I was going to add orgs as a param to this test